### PR TITLE
Manage object lists (update, delete list, delete list items)

### PR DESCRIPTION
**Pull request by barbarah**
Based on design: https://gui-prototype.colonialcollections.nl/community-object-list.html

### CODE REVIEW

File: apps/researcher/src/app/[locale]/communities/[slug]/[listId]/actions.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import {objectList} from '@colonial-collections/database';
+import {revalidatePath} from 'next/cache';
+
+interface Props {
+  id: number;
+  revalidatePathName: string;
+}
+
+export async function deleteObjectFromList({id, revalidatePathName}: Props) {
+  await objectList.removeObject(id);
```
Comment (sdevalk) [inline apps/researcher/src/app/[locale]/communities/[slug]/[listId]/actions.ts:12]: For consistency, use the same terms? The function uses `delete...` whereas the function from `objectList` uses `remove...`.

File: apps/researcher/src/app/[locale]/communities/[slug]/[listId]/buttons.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+export function DeleteObjectButton({id}: DeleteObjectButtonProps) {
+  const t = useTranslations('ObjectList');
+  const pathName = usePathname();
+  async function deleteObjectFromListClick() {
+    try {
+      await deleteObjectFromList({id, revalidatePathName: pathName});
+    } catch (err) {
+      console.error(err);
```
Comment (sdevalk) [inline apps/researcher/src/app/[locale]/communities/[slug]/[listId]/buttons.tsx:21]: Should we notify the user if the deletion was unsuccessful?

File: apps/researcher/src/app/[locale]/communities/[slug]/[listId]/actions.ts (hunk 1 above)
Comment (sdevalk) [inline apps/researcher/src/app/[locale]/communities/[slug]/[listId]/actions.ts:12]: For consistency, use either `removeObjectFromList()` or `deleteObject()`?

Comment (barbarah) [reply apps/researcher/src/app/[locale]/communities/[slug]/[listId]/actions.ts:12]: Sorry, I see I only changed a different action.

**Review by sdevalk — APPROVED** [review]

**Pull request merged**